ch08-3Dgraphics/hello3d/pom.xml

```diff
@@ -29,7 +29,7 @@
             <plugin>
                 <groupId>org.openjfx</groupId>
                 <artifactId>javafx-maven-plugin</artifactId>
-                <version>0.0.1</version>
+                <version>0.0.3</version>
                 <configuration>
                     <mainClass>org.modernclientjava.hello3d.Shapes</mainClass>
```

Comment:
Rename `hello3d` to `javafx3d`?

Reply:
that particular sample is not used in the book, and I guess we just have to remove it.